pkg/virt-api/api.go

```diff
@@ -74,6 +74,9 @@ const (
 	defaultTlsKeyFilePath      = "/etc/virt-api/certificates/tls.key"
 	defaultHandlerCertFilePath = "/etc/virt-handler/clientcertificates/tls.crt"
 	defaultHandlerKeyFilePath  = "/etc/virt-handler/clientcertificates/tls.key"
+
+	notFound   = "Not Found"
+	badRequest = "Bad Request"
 )
```

Comment:
Maybe somehow indicate this is a string/message/label it's not so clear when it's used (`Returns(http.StatusNotFound, notFound, "")` )  ?

Reply:
+1

It should tell what it is. E.g: `httpStatusBadRequestMessage`